CIPs/0006.md

```diff
@@ -0,0 +1,48 @@
+# CIP [0006]: Implement ElectionVote, GovernanceUpvote and GovernanceVote functions in releasegold contract
+
+- Date: 2020-05-03
+- Author: @zviadm
+- Status: DRAFT
```

Comment:
Hi @zviadm thank you so much for submitting a Celo CIP!

Can you update your draft to follow the new CIP Template here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Once the missing items are added and it follows the new template format, I'll merge the draft. Let me know if you'd like me to help you out on guidance on the new template.

Reply:
This CIP is no longer relevant, since most of the ReleaseGold owners have figured out how to manage their locking and voting. And there isn't huge amounts of new ReleaseGold contracts issued anymore. 
Thus we can just close this CIP